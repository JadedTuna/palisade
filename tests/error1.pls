in {}
out {}

num := 20;
b := false;
if (num >= b) {
    // cannot compare int to bool
    num = 0;
}
