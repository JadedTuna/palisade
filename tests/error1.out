[1;31merror: [0mtype mismatch
   1 | high num = 20;
   2 | high bool = false;
   3 | if(num [1;31m>=[0m bool) {
       [1;31m~~~~~~~[0m[1;31m^^[0m
